src/ControlSystem/WriteData.hpp

```diff
@@ -33,6 +33,7 @@ namespace control_system {
  * - d2tLambda
  * - ControlError
  * - dtControlError
+ * - Timescale
  *
  * where "Lambda" is the function of time value.
```

Comment:
Why don't we just call it "FunctionOfTime" and "dtFunctionOfTime" in the output? Then you don't need to document what it is.

Reply:
#4827 I'll do this in a follow up PR